pkgs/applications/video/gnome-mpv/default.nix

```diff
@@ -4,22 +4,25 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "gnome-mpv-${version}";
-  version = "0.13";
+  pname = "gnome-mpv";
+  version = "0.16";
 
   src = fetchFromGitHub {
-    owner = "gnome-mpv";
-    repo = "gnome-mpv";
-    rev = "0d73b33d60050fd32bf8fae77d831548970a0b69"; # upstream forgot to update appdata
-    # rev = "v${version}";
-    sha256 = "1cjhw3kz163iwj2japhnv354i1lr112xyyfkxw82cwy2554cfim4";
+    owner = pname;
```

Comment:
I think we should just switch to `"celluloid-player"` & `"celluloid"` here since the repo was already renamed.

Reply:
I debated on renaming the whole package actually. On the website it's mentioned that they've renamed, but for now the binary is still called `gnome-mpv` and in the [release notes for 0.16](https://github.com/celluloid-player/celluloid/releases) it says:

> From the next release onwards GNOME MPV will be renamed to Celluloid. The repository will be moved to https://github.com/celluloid-player/celluloid.

So it seems like it's still gnome-mpv, at least somewhat officially. But I don't mind too much if you want to use the new repo. Thoughts?

Reply:
Yeah, 0.16 is still gnome-mpv, 0.17 (or whatever the next version is going to be) will be celluloid. So I would keep the package name and attribute for now.

Using `pname` here will not work once we change it to celluloid, since the owner and repo are different. Changing the attributes to "celluloid-player" & "celluloid" will work now and after the rename as well.

Reply:
Done.